BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -119,8 +119,12 @@ - (instancetype)initWithURL:(NSURL *)URL HTTPMethod:(BOXAPIHTTPMethod *)HTTPMeth
         NSMutableURLRequest *APIRequest = [NSMutableURLRequest requestWithURL:[self requestURLWithURL:_baseRequestURL queryStringParameters:_queryStringParameters]];
         APIRequest.HTTPMethod = HTTPMethod;
 
-        NSData *encodedBody = [self encodeBody:_body];
-        APIRequest.HTTPBody = encodedBody;
+        if (_body != nil) {
```

Comment:
The POSTParams could allocate and empty dictionary. Perhaps we can add that check inside encodeBody and return nil.